### ISSUE TITLE
Background color of first and last page (help needed)

### ISSUE THREAD

**Issue by rafaelrsss** [rstudio/pagedown#180]
Hi all.
I wanted to change the background color only of the first and last page of my report. Is this possible? I would also like to change the font color of the first page to white.

Any help is appreciated.

**Comment by RLesur** [comment]
Hi @rafaelrsss,

Sure, this is possible. Can you provide some information about your context?  
What output format do you use?  
Are you using some custom CSS?

The answer may be different depending on the context.

Here's an example that should work in most of cases:

    ---
    title: "A Multi-page HTML Document"
    author: "Yihui Xie and Romain Lesur"
    date: "`r Sys.Date()`"
    output:
      pagedown::html_paged:
        toc: true
        # change to true for a self-contained document, but it'll be a litte slower for Pandoc to render
        self_contained: false
    # uncomment this line to produce HTML and PDF in RStudio:
    #knit: pagedown::chrome_print
    ---
    
    ```{css}
    .pagedjs_first_page {
      background-color: #a28089;
      color: white;
    }
    
    .pagedjs_page:last-of-type {
      background-color: #8458b3;
      color: white;
    }
    ```
    
I cannot guarantee you that this answer will work in your context (if you already use custom CSS for example).

**Comment by rafaelrsss** [comment]
Hi, thanks it works.
I am using html_paged to get an automatic report in pdf for wind analysis.

I tried to create a custom.css and load together with `css: ["default-fonts", "default-page", "default"]` but it was not working... I dont know why, probably I did something wrong. But the way you suggested is working so I will keep like that.

The very last page is still white, but I think I saw an explanation regarding this mentioning that it was designed this way to simulate a book layout. I will try to solve this post processing the pdf and removing this page "by force".

Thanks a million!
